src/util/std_types.h

```diff
@@ -590,34 +537,27 @@ class union_tag_typet:public tag_typet
   }
 };
 
-/*! \brief Cast a generic typet to a \ref union_tag_typet
- *
- * This is an unchecked conversion. \a type must be known to be \ref
- * union_tag_typet.
- *
- * \param type Source type
- * \return Object of type \ref union_tag_typet
- *
- * \ingroup gr_std_types
-*/
+/// \brief Cast a generic typet to a \ref union_tag_typet
+///
+/// This is an unchecked conversion. \param type must be known to be \ref
+/// union_tag_typet.
+///
+/// \param type Source type.
+/// \return Object of type \ref union_tag_typet.
 inline const union_tag_typet &to_union_tag_type(const typet &type)
 {
   PRECONDITION(type.id()==ID_union_tag);
   return static_cast<const union_tag_typet &>(type);
 }
 
-/*! \copydoc to_union_tag_type(const typet &)
- * \ingroup gr_std_types
-*/
+/// \copydoc to_union_tag_type(const typet &)
 inline union_tag_typet &to_union_tag_type(typet &type)
 {
   PRECONDITION(type.id()==ID_union_tag);
   return static_cast<union_tag_typet &>(type);
 }
 
-/*! \brief A generic enumeration type (not to be confused with C enums)
-*/
-
+/// A generic enumeration type (not to be confused with C enums)
```

Comment:
What is it used for?

Reply:
No idea, @smowton doesn't know either. Any ideas? It has a list of elements and is only used in casts and direct checks for the type, not clear then what it should represent.

Reply:
It seems it predates Subversion/git history. @kroening? I'd be inclined to create an issue suggesting its removal...